### DISCUSSION TITLE
VsCode extension samples and tutorials do not get loaded by extension host

### DISCUSSION THREAD

**Discussion by ac10n** [microsoft/vscode-discussions#112] [Extension Development QnA] [answered]
I'm developing the [Taqueria extension](https://marketplace.visualstudio.com/items?itemName=ecadlabs.taqueria-vscode), [source code](https://github.com/ecadlabs/taqueria/tree/main/taqueria-vscode-extension). I wanted to subscribe to notifications from a SignalR service, so I followed the instructions in SignalR, but I got the error `requireFunc is not a function`. I asked it [here](https://stackoverflow.com/questions/73463906/signalr-does-not-work-in-a-vscode-extension), but got no answer.

Then I decided to do the same in a minimal extension. So I followed the instructions in VsCode extension development docs to generate a simple extension (which is basically running `yo code`). But when I click debug, the new window that opens (Extension host), does not load the extension I have created. I tried every output window in both VsCode windows, and I got no error or warning that helps diagnose the problem.

I cloned the VsCode extension samples repo and same happens with the "Hello-World" sample.

Some months ago I had success with both the tutorial and with samples.

I have followed the process on an m1 mac, as well as a Fedora Linux.

So, here are my questions:

1. How can I add SignalR to a VsCode extension?
2. Why the sample and tutorial no longer work?
3. What debug tools do we have when the extension does not load? Are there additional debug outputs that are not trivial to find?

**Comment by ac10n** [comment]
The code that starts SignalR and fails can be seen in [this draft PR](https://github.com/ecadlabs/taqueria/pull/1187)

**Comment by burkeholland** [comment, marked as answer]
It looks like SignalR has had some trouble with WebPack in the past as it loads some things dynamically. Could this be related to that issue? https://github.com/dotnet/aspnetcore/issues/21864
